images/macos/toolsets/toolset-13.json

```diff
@@ -3,7 +3,7 @@
         "default": "14.2",
         "x64": {
             "versions": [
```

Comment:
if we pin VersionOS version it should be done in a different part of toolset so we could call it like `Xcode.Versionos.Version` in jq, lets pull it out

Reply:
it makes no sense in different part of a toolset, because it is tightly paired with XCode